### ISSUE TITLE
allow downloading of message by it's id

### ISSUE THREAD

**Issue by jludvice** [czgov/camel-isds#30]
It might be used for instance like this:
```java
// camel route example
from("some event")
   .header("id", "45678")
   .to("isds:download?username=asdf&password=asdf")
```

**Comment by jludvice** [comment]
There is catch with downloading received message by it's ID.

When somebody receives message, it's in state received.
Once he logs into databoxes via browser, or calls list of rceceived messages method via API,
the message changes state into user logged in.

It's not possible to download message until it's in this "user logged in" (or nother delivered )state.

![docs](https://user-images.githubusercontent.com/8707241/29970366-8ea753b6-8f24-11e7-9fcf-ee0007d18eca.png)

**Comment by jludvice** [comment]
Need new version of java-isds with fix for czgov/java-isds#17
